roombapy/remote_client.py

```diff
@@ -71,12 +71,15 @@ def connect(self):
                 self._open_mqtt_connection()
                 return True
             except Exception as e:
+                _last_error = "Can't connect to {}, error: {}".format(self.address, e)
```

Comment:
Why not store this as a class attribute?

Reply:
As it's only needed during connect and the value is passed to the caller by on_connect, I think it's better to leave it local.